plugins/win-capture/graphics-hook/CMakeLists.txt

```diff
@@ -29,10 +30,10 @@ target_sources(
             graphics-hook.rc)
 
 target_compile_definitions(_graphics-hook INTERFACE COMPILE_D3D12_HOOK)
-target_compile_options(_graphics-hook INTERFACE $<IF:$<CONFIG:Debug>,/MTd,/MT>)
```

Comment:
No longer needed?

Reply:
See above, I don't encounter any compile errors with this removed. This switch was usually added because of runtime library mismatches.

It could have been implicitly fixed by `ipc-util` having become an interface library and thus its sources are compiled with the actual target using the same runtime.